bulletroboy/roboy_simulation.py

```diff
@@ -0,0 +1,76 @@
+import argparse
+import os
+
+import pybullet as p
+import math
+import time
+
+import rclpy
+
+from threading import Thread
+from bulletroboy.roboy import BulletRoboy
+from bulletroboy.environment_control import EnvironmentCtrl
+
+def is_valid_file(parser, arg):
+    file_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
@NadaChatti m? how does it fix #10? `file_path` is still prepended before the argument

Reply:
We do not use that method for paths given explicitly anymore

Reply:
why not? you still want to check if the file exists and valid?

Reply:
Yes you're right, I changed it.